src/search.rs

```diff
@@ -169,6 +169,17 @@ impl BlindIndexSearch {
         )
         .map_err(|message| IronOxideErr::ValidationError("data".to_string(), message))
     }
+
+    /// Process a string using the same transliteration that is done when a string is processed to
+    /// tokenize data or a query. This processing will drop any characters that are ignored (mostly
+    /// punctuation), remove any diacritical marks, convert characters to sequences of latin equivalents,
+    /// and lower case all characters.
+    /// For example, the string “Æneid” is converted to “aeneid”, and “北亰” is converted to “bei jing ”.
+    ///
+    /// s - The string you want to process
+    pub fn transliterate_string(&self, s: &str) -> String {
```

Comment:
Seems like since this doesn't use BlindIndexSearch that we should instead just `pub use` the `transliterate_string` and allow it to be exposed out in the search module?

Reply:
Yup - fixed and made sure the docs retrieved the info from search helpers for the function.